store/urls.py

```diff
@@ -4,6 +4,6 @@
 app_name = "store"
 
 urlpatterns = [
-    path('book/', views.books_list),
+    path('books/', views.books_list),
     path('book/<int:pk>/', views.book_detail,name='book'),
```

Comment:
```suggestion
    path('books/<int:pk>/', views.book_detail,name='book'),
```
Cambiar esto a plural para que quede consistente con el otro endpoint